plugins/restapi/v2/utxo.go

```diff
@@ -85,7 +89,23 @@ func outputByID(c echo.Context) (*OutputResponse, error) {
 		}
 		return nil, errors.WithMessagef(echo.ErrInternalServerError, "reading output failed: %s, error: %s", outputID.ToHex(), err)
 	}
-	return NewSpentResponse(spent, ledgerIndex)
+	return NewSpentResponse(spent, ledgerIndex, metadataOnly)
+}
+
+func rawOutputByID(c echo.Context) ([]byte, error) {
+	outputID, err := restapi.ParseOutputIDParam(c)
+	if err != nil {
+		return nil, err
+	}
+
+	bytes, err := deps.UTXOManager.ReadRawOutputBytesByOutputIDWithoutLocking(outputID)
+	if err != nil {
+		if errors.Is(err, kvstore.ErrKeyNotFound) {
+			return nil, errors.WithMessagef(echo.ErrNotFound, "output not found: %s", outputID.ToHex())
+		}
```

Comment:
There is an error return missing here in case its another error than `ErrKeyNotFound`